Code deprecation errors

```
Error: Usage of deprecated trait Drupal\Tests\field\Traits\EntityReferenceTestTrait in class Drupal\Tests\localgov_news\FunctionalJavascript\PromotedNewsWidgetTest:
in drupal:10.2.0 and is removed from drupal:11.0.0. Use
   \Drupal\Tests\field\Traits\EntityReferenceFieldCreationTrait instead.
 ------ ------------------------------------------------------------------------- 
  Line   tests/src/FunctionalJavascript/PromotedNewsWidgetTest.php                
 ------ ------------------------------------------------------------------------- 
  19     Usage of deprecated trait                                                
         Drupal\Tests\field\Traits\EntityReferenceTestTrait in class              
         Drupal\Tests\localgov_news\FunctionalJavascript\PromotedNewsWidgetTest:  
         in drupal:10.2.0 and is removed from drupal:11.0.0. Use                  
            \Drupal\Tests\field\Traits\EntityReferenceFieldCreationTrait          
         instead.                                                                 
 ------ ------------------------------------------------------------------------- 


Error:  [ERROR] Found 1 error              
```

--- a/tests/src/FunctionalJavascript/PromotedNewsWidgetTest.php
+++ b/tests/src/FunctionalJavascript/PromotedNewsWidgetTest.php
@@ -16,6 +16,9 @@ use Drupal\Tests\node\Traits\NodeCreationTrait;
 class PromotedNewsWidgetTest extends WebDriverTestBase {
 
   use ContentTypeCreationTrait;
+  // FIXME: Replace with EntityReferenceFieldCreationTrait when Drupal 10.1 is
+  // end of life.
+  // @phpstan-ignore-next-line.
   use EntityReferenceTestTrait;
   use NodeCreationTrait;
 

